constructor/winexe.py

```diff
@@ -86,6 +86,23 @@ def insert_tempfiles_commands(paths: os.PathLike) -> List[str]:
         lines.append(f"File {path}")
     return lines
 
+def setup_extra_env_variables(info) -> List[str]:
+    """Helper function to insert extra environment variables into nsis template.
+
+    Args:
+        info: Dictionary of information parsed from construct.yaml
+
+    Returns:
+        List[str]: Commands to be inserted into nsi template
+    """
+    lines = []
+    for variable in info.get('extra_env_variables',[]):
+        split = variable.split('=')
+        if len(split)==2:
+            (name, val) = split
+        lines.append(f"""System::Call 'kernel32::SetEnvironmentVariable(t,t)i("{name}", "{val}").r0'""")
```

Comment:
```suggestion
        lines.append(f"""System::Call 'kernel32::SetEnvironmentVariable(t,t)i("{name}", {str_esc(value)}).r0'""")
```

We need to be careful with values that have special characters. `str_esc` helps a bit, and also adds the quotes, so this is a bit safer.